plugins/todo-backend/src/lib/TodoReader/TodoScmReader.ts

```diff
@@ -167,3 +172,26 @@ export class TodoScmReader implements TodoReader {
     return { result: { items: todos }, etag: tree.etag };
   }
 }
+
+/**
+ * @alpha
+ */
+export const todoScmReaderRef = createServiceRef<TodoScmReader>({
+  id: 'todoScmReaderFactory',
```

Comment:
```suggestion
export const todoReaderServiceRef = createServiceRef<TodoReader>({
  id: 'todo.todoReader',
```

The type of this one should be the contract (interface), not the concrete class. And bear in mind [the naming](https://github.com/backstage/backstage/blob/17969f1d70b7e9257e31672fe4f757ffc9f31157/docs/backend-system/architecture/07-naming-patterns.md?plain=1#L83) patterns.